cmd/kubemark/hollow-node.go

```diff
@@ -211,9 +212,20 @@ func run(config *hollowNodeConfig) {
 			WithTraceDisabled: true,
 		}
 
+		arktosExtClientConfig := *clientConfigs
```

Comment:
if possible, we should reuse the client created in Arktos instead of generating one here. some core Arktos code was used in here is concerning.

Reply:
this config cannot be reused; intentionally has a copy here - as it has different content type from regular core kube-client.

Reply:
I meant to say reuse the clientset: i.e. staging/src/k8s.io/arktos-ext/pkg/generated/clientset/versioned/typed/arktosextensions/v1/arktosextensions_client.go so it won't need to be build in here in hollow-node code.

Reply:
agreed; will address the code reuse later.